### ISSUE TITLE
Cannot see images in VS 2022

### ISSUE THREAD

**Issue by beccamc** [dotnet/machinelearning-modelbuilder#1785]
Can't see images. Looks like something is wrong and they're null, also creating an accessibility problem. 

VS 17.0.0 Preview 5 [31710.301 Main]
MB 16.7.4.2146402

### Can't see images

![image](https://user-images.githubusercontent.com/9122518/133327113-e85d48a2-cb89-4a22-8a91-df04d21ed104.png)

### Section 508 502.3.1: (devenv/image '') An onscreen element must not have a null BoundingRectangle property.

The following accessibility issue needs investigation.

**App:** devenv

**Element path:** image ''

**Issue Details:** An onscreen element must not have a null BoundingRectangle property. [Section 508 502.3.1](https://www.access-board.gov/ict/#502-interoperability-assistive-technology)

**How To Fix:** If the element is offscreen, set its IsOffscreen property to TRUE.
If the element is onscreen, provide a BoundingRectangle property.

This accessibility issue was found using Accessibility Insights for Windows, a tool that helps debug and find accessibility issues earlier. Get more information and download this tool at https://aka.ms/AccessibilityInsights.

**Comment by beccamc** [comment]
This is also happening on the "Next steps" page

![image.png](https://images.zenhubusercontent.com/5d55c3d68fac840001314eaa/de25997b-fd61-4b87-b344-219ade67ae8f)

**Comment by zewditu** [comment]
@LittleLittleCloud  this has also happened in preview 4 and you had an email thread with Vs extensibility team and they recommended running the following command 
"
devenv.exe /clearCache

devenv.exe /updateConfiguration

devenv.exe /resetUserData
"
I had a similar issue a week ago  this helps me to fix 
But now I test this one on preview 5 it does not work.
is this a similar issue you discuss with that email thread?

**Comment by beccamc** [comment]
This is fixed with a VS update. 
![image.png](https://images.zenhubusercontent.com/5d55c3d68fac840001314eaa/686f2393-e9c9-4582-addd-d936ceeb005e)

If this problem comes up again we should bring it up to the framework team.
